tests/cross/src/test/scala/tests/pc/CompletionCaseSuite.scala

```diff
@@ -402,7 +406,7 @@ class CompletionCaseSuite extends BaseCompletionSuite {
   )
 
   check(
-    "brace-negative",
+    "brace-negative".tag(IgnoreScala3),
```

Comment:
Could you write what is the issue here? Maybe todo plus create an issue in the repo?

Reply:
The issue there was that this was not a case completion but a named argument one and was probably here to check if not suggest case completions. Maybe we can just add compat for scala3 instead of ignoring it?

Reply:
It's better to add compat so that reviewers can see easier what is happening.